src/web/server/locales/en/validation.json

```diff
@@ -8,16 +8,15 @@
   "missingFirstName": "Enter your first or given name",
   "missingLastName": "Enter your last or family name",
   "selectYesOrNo": "Select yes or no",
+  "selectYesOrNoAreYouPregnant": "Select yes if you’re pregnant",
   "selectYesOrNoDoYouLiveInScotland": "Select if you live in Scotland or not",
   "selectYesOrNoDoYouHaveChildrenThreeOrYounger": "Select if you have any children who are three years old or younger",
   "invalidPostcode": "Enter a correct postcode, like AA1 1AA",
   "informationTooLong": "The information you entered is too long",
   "missingAddressField": "Tell us your address",
-  "expectedDeliveryDateInvalid": "Enter your baby's due date",
-  "expectedDeliveryDateTooFarInPastTitle": "The due date you entered is more than 1 month ago",
-  "expectedDeliveryDateTooFarInPast": "Please call our helpline on 0345 607 6823 to talk about your application.",
-  "expectedDeliveryDateTooFarInFutureTitle": "The date you have entered is more than 8 months in the future. You must be at least 10 weeks pregnant to apply for yourself.",
-  "expectedDeliveryDateTooFarInFuture": "If you have children under the age of 4, answer ‘no’ to this question to continue with this application on their behalf.",
+  "expectedDeliveryDateInvalid": "Enter the due date in the correct format",
+  "expectedDeliveryDateTooFarInPast": "Enter a due date to show you’re at least 10 weeks pregnant",
+  "expectedDeliveryDateTooFarInFuture": "Enter a due date to show you’re at least 10 weeks pregnant",
```

Comment:
I don't understand how this will now work without changes to the njk file too, perhaps I'm missing something. I remember having to do this slightly differently for this scenario, hence why we needed the extra entries.